nextras-orm-generator/composer.json

```diff
@@ -0,0 +1,5 @@
+{
+  "require": {
+    "contributte/nextras-orm-generator": "dev-master"
```

Comment:
Use branch alias for dev-master please, so it cannot broke in future

Reply:
as soon as I release v0.2 (tomorrow ) I can also change this to ^0.2

Reply:
Yep. I've merged documentation PR, the last thing is release v0.2. Let's do it. :-)